cabal.project

```diff
@@ -1,6 +1,23 @@
--- Bump this if you need newer packages.
+-- Custom repository for cardano haskell packages
```

Comment:
Can we add the following?

See https://github.com/input-output-hk/cardano-haskell-packages on how to use CHaP in a Haskell project.

Reply:
:+1: